### ISSUE TITLE
Doesn't work with multiple domains

### ISSUE THREAD

**Issue by greenec** [Daplie/letsencrypt-cli#12]
Here is my configuration. This generates certs, but only for the latter domain parameter listed.
```
sudo letsencrypt certonly \
  --agree-tos --email connor.greene2000@gmail.com \
  --standalone \
  --domains www.chessmatch.us \
  --domains chessmatch.us \
  --config-dir /etc/letsencrypt
```
The line below results in a crash.
```
--domains www.chessmatch.us,chessmatch.us
```

**Comment by rubenvereecken** [comment]
Can confirm. It only happens with multiple domains.

**Comment by cconley414** [comment]
Yep, confirmed as well.

**Comment by coolaj86** [comment]
I've got an update to `node-letsencrypt` coming in a few days that should fix this I think.

**Comment by coolaj86** [comment]
This is finally fixed in v2.0.0, which was just published.
https://github.com/Daplie/letsencrypt-cli

Enjoy!

Note: you must always specify the domains with the primary listed first:

For example, if you specify `--domains example.com,www.example.com`, you can renew with just `--domains example.com`, but you cannot renew with `--domains www.example.com,example.com` as it will look in the wrong place for the file.
